LESS/form.less

```diff
@@ -1,7 +1,66 @@
 // FORM SECTION
 
 .form-section {
-	border: 1px solid black;
-	background-color: @green;
-	padding: 30px 10px;
+	background: linear-gradient(rgba(76, 187, 23, .8), rgba(76, 187, 23, .8)), url("https://bit.ly/2HS7yUF");
+	color: white;
+	padding: 60px 10px;
+	.flex-align(column, stretch);
+	text-align: center;
+	p {
+		padding: 10px 0 20px 0;
+	}
+	form {
+		text-align: left;
+		font-family: @text-font;
+		.flex-align(column, flex-end);
+		@media @desktop {
+			padding: 10px 80px;
+			.flex-align(row, center);
+			flex-wrap: wrap;
+			justify-content: space-between;
+		}
+		input[type=text],
+		textarea {
+			width: 100%;
+			background-color: transparent;
+			border: 1px solid white;
+			padding: 15px 0;
+			margin-bottom: 15px;
+			&:focus {
+				background-color: white;
+			}
+		}
+		input[type=text] {
+			@media @desktop {
+				width: 48%;
+			}
+		}
```

Comment:
make the font of the input field bigger. In my opinion the font is too small for the input field